src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/prison/PhoneDetail.kt

```diff
@@ -4,5 +4,5 @@ data class PhoneDetail(
   val phoneId: Int,
   val number: String,
   val type: String,
-  val ext: String,
+  val ext: String?,
```

Comment:
probably need to check the nullability of these

Reply:
I've been through and tried to match these - hopefully we will have a clearer picture once we test in dev!